pydatastructs/miscellaneous_data_structures/tests/test_segment_trees.py

```diff
@@ -0,0 +1,87 @@
+import random
+from pydatastructs.miscellaneous_data_structures.segment_tree import SegmentTree
```

Comment:
That's here the problem is. Look for `__all__` variables. Try to observe the pattern on how imports happen via, `from pydatastructs import some_stuff`.

Reply:
Alright...